migration/tests.py

```diff
@@ -447,4 +454,7 @@ def test_main_migrates_tool_successfully(self):
     httpcore_level = logging.getLogger('httpcore')
     httpcore_level.setLevel(http_log_level)
 
+    asyncio_logger = logging.getLogger('asyncio')
+    asyncio_logger.setLevel(logging.ERROR)
```

Comment:
I was getting a lot of log warnings from `asyncio` without this, mostly I think because we're using a number of synchronous processes next to the asynchronous ones and it's warning us that some tasks are taking a long time.

Reply:
Make sense. Should we have the same setting inside manager.py?

Reply:
I'm not seeing this error with `trio`, for whatever reason, but the test cases use `asyncio`.